sdk/ml/azure-ai-ml/azure/ai/ml/constants/_compute.py

```diff
@@ -42,5 +42,11 @@ class CustomApplicationDefaults:
     ENDPOINT_NAME = "connect"
 
 
+class ComputeSizeTier:
+    AML_COMPUTE_DEDICATED = "amlComputeDedicatedVMSize"
+    AML_COMPUTE_LOWPRIORITY = "amlComputeLowPriorityVMSize"
+    COMPUTE_INSTANCE = "computeInstanceVMSize"
```

Comment:
Please add docstring for the attributes of this ComputeSizeTier class.

Reply:
This class is a set of constants -- we haven't provided docstrings for them in the past, so I don't think that would be applicable here.

Reply:
Ended up adding this after an offline discussion.